src/column.rs

```diff
@@ -850,29 +857,112 @@ impl HashColumn {
 		Ok(())
 	}
 
-	fn dump(&self, log: &Log, check_param: &crate::CheckOptions, col: ColId) -> Result<()> {
-		let start_chunk = check_param.from.unwrap_or(0);
-		let end_chunk = check_param.bound;
+	fn iter_index_fast(
+		&self,
+		log: &Log,
+		mut f: impl FnMut(IterStateOrCorrupted) -> Result<bool>,
+		_start_chunk: u64,
+	) -> Result<()> {
+		let tables = self.tables.read();
+		let index = &tables.index;
+
+		let entries = index.sorted_entries()?;
```

Comment:
Could probably use start_chunk as a restart parameter, but would still need to rebuild sorted entries at all restart so probably something that only make sense to put in place if really needed at some point.